src/main/java/uk/gov/hmcts/ccd/domain/casestate/EnablingConditionSorter.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.ccd.domain.casestate;
+
+import java.util.Comparator;
+import java.util.List;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.domain.model.definition.EventPostStateDefinition;
+
+@Component
+public class EnablingConditionSorter {
```

Comment:
EnablingConditionPrioritiser?

Reply:
Changed to **PrioritiseEnablingCondition**